crates/swc_ecma_transforms_module/src/module_ref_rewriter.rs

```diff
@@ -101,36 +101,42 @@ impl VisitMut for ModuleRefRewriter {
     fn visit_mut_function(&mut self, n: &mut Function) {
         n.params.visit_mut_with(self);
```

Comment:
param is not top level.
[babel](https://babeljs.io/repl#?browsers=chrome%2055&build=&builtIns=entry&corejs=3.6&spec=false&loose=false&code_lz=KYDwDg9gTgLgBAMwK4DsDGMCWEWIhACgEM4BeOGAC0wGcBKOAbzRxogBtgA6diAc2J0AvgCggA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=env&prettier=false&targets=&version=7.18.7&externalPlugins=&assumptions=%7B%7D)

Reply:
Thanks

Reply:
visit_mut_class_method could be removed now.